overlays/cross-compilation/papirus-icon-theme/default.nix

```diff
@@ -0,0 +1,11 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# papirus-icon-theme cross-compilation fixes (removing qt dependency)
```

Comment:
Was this done because qt doesn't cross-compile just now or to make the build process faster?

Reply:
cc @humaidq-tii